lib/account_deleter.rb

```diff
@@ -17,8 +17,8 @@ class AccountDeleter
 
   attr_accessor :person, :user
 
-  def initialize(diaspora_handle)
-    self.person = Person.where(:diaspora_handle => diaspora_handle).first
+  def initialize(person)
```

Comment:
The change is not critical, I can revert it. But what's wrong with passing object here? It's not worker enqueueing here, so it's not passed via Redis, right?

I just thought, that in all the usages of the AccountDeleter (currently one, more in #6750) the person object is already available in some way and there is no point searching it via diaspora_handle.

For example, [here](https://github.com/diaspora/diaspora/pull/6783/files#diff-0754ce5ce1ca26e1c990fa947f873486R33) person is linked via belongs_to relationship, and perhaps it is found faster than a search via diaspora_handle. But that's a pure speculation.